controllers/users.js

```diff
@@ -600,6 +602,55 @@ const setInDiscordScript = async (req, res) => {
   }
 };
 
+/**
+ * Collects all Users with ONBOARDING state and are present in discord server for more than 31 days
+ *
+ * @param req {Object} - Express request object
+ * @param res {Object} - Express response object
+ */
+
+const getUsersWithOnboardingState = async (req, res) => {
+  let minDaysInServer = 31;
+  try {
+    const { allUserStatus } = await userStatusModel.getAllUserStatus(req.query);
+    const { minPresenceDays } = req.query;
```

Comment:
Nit:- minPresenceDays=>daysCount or numberOfDays

Giving idea...please ask in discord also:)